### PR TITLE
Add recipe for ldap package

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

**Comment by nicoddemus** [comment]
@conda-forge/staged-recipes @ocefpaf I believe this is ready for review/merge.

File: recipes/ldap/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+test:
+  imports:
+    - ldap
+  commands:
+    - ldap --help
+
+about:
+  home: https://github.com/andreif/ldap
+  license: BSD
```
Comment (ocefpaf) [inline recipes/ldap/meta.yaml]: 1, 2, or 3 :wink:

**Comment by ocefpaf** [comment]
The license seems to be BSD-3 and I added the SPDX short version for it.

**Comment by nicoddemus** [comment]
Awesome, thanks @ocefpaf!

**Pull request merged**